builder/googlecompute/builder_acc_test.go

```diff
@@ -328,18 +348,21 @@ func TestAccBuilder_ImageArchVariations(t *testing.T) {
 	}
 
 	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			imageName := fmt.Sprintf("%s-%d", tt.name, time.Now().UTC().Unix())
+		testRun := tt
```

Comment:
Why do you create this separate object? Can't you just use tt everywhere?

Reply:
Ah, very good question! Because the test runs in parallel, the captured object will be replaced by the next item, so I need to capture it locally with this in order for the value to be what it was when the coroutine is created.
I would've been caught by this tbh, `go vet` saved me there by pointing it out